app/src/test/java/java_server/RequestHandlerTest.java

```diff
@@ -0,0 +1,35 @@
+package java_server;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+
+public class RequestHandlerTest {
+    private InputStream inputStream;
+    private OutputStream byteArrayOutputStream;
+//    private MockSocket mockSocket;
+    private RequestHandler requestHandler;
+
+    @Before
+    public void setUp() throws Exception {
+        inputStream = new ByteArrayInputStream("GET / HTTP/1.1".getBytes());
+        byteArrayOutputStream = new ByteArrayOutputStream();
+//        mockSocket = new MockSocket(inputStream, byteArrayOutputStream);
+//        requestHandler = new RequestHandler(mockSocket);
```

Comment:
evita hacer mock de cosas que no son tuyas – pues terminas implementando detalles que alguien más invento

https://markhneedham.com/blog/2009/12/13/tdd-only-mock-types-you-own/